### ISSUE TITLE
What was wrong with old colors & styling?

### ISSUE THREAD

**Issue by andreif** [rowanj/gitx#338]
Hi, I have recently updated GitX and just cannot hold my frustration anymore. Instead of all the issues that do need to be addressed you guys decided to make some decorative changes which just made reading the tree harder and annoying (I guess only for me?). Font letter spacing is off so two identical lines don't have the same width. And do you really needed shadows? Anyway, I don't feel like using GitX anymore, which is sad because I was a huge fan of it. Great job for pushing me to the command line :+1:  and thanks for all the former releases.

**Comment by ngrilly** [comment]
I agree that the decorative changes could not be to the taste of everyone, but I disagree with your conclusion. They are a lot of useful improvements in this new version, like for example the colored word diff. I think the hard work of @rowanj and contributors deserves better than your message. We're lucky to have people like them taking the time to build tools for all of us. If you want to convince @rowanj to revert back to the previous style, then find the adequate arguments.

**Comment by rowanj** [comment]
:thumbsup:

**Comment by ghost** [comment]
It a question of taste, I like a lot the color scheme of the last official release, for me a lot more readable than previous one... 

No need to comment the aggressive color of the first commentary... It's just a shame...
